src/main/java/core/basesyntax/utility/service/ReportService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.utility.service;
```

Comment:
I'm sorry, thats because first I made the util classes with a static methouds. But on Q&A was discussed that, and I replaced this in code, but don't managed packages after that